extensions/ohmyzsh-git-alias/src/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { Action, ActionPanel, List } from "@raycast/api";
+import { useState } from "react";
+import { default as alias } from "./alias.json";
+
+export default function Command() {
+  const [searchText, setSearchText] = useState<string>("");
+
+  const onSearchTextChange = (text: string) => {
+    setSearchText(text);
+  };
+
+  return (
+    <List onSearchTextChange={onSearchTextChange}>
+      {alias.map((a: { alias: string; command: string }) => {
+        if (searchText !== "" && !a.alias.includes(searchText) && !a.command.includes(searchText)) {
```

Comment:
Is there a reason you are checking for this? Raycast should handle search manually pretty well!

Reply:
It looks like raycast can't search for subtitle.

Reply:
you can use the `keywords` prop, the search takes that into account and is hidden from the UI. That way you can still rely on fuzzy search :)

Reply:
https://developers.raycast.com/api-reference/user-interface/list#list.item

Reply:
Awesome. Thank you.